src/Module/Util/VaInfo.php

```diff
@@ -316,6 +316,36 @@ public function write_id3($tagData)
         $tagWriter->tag_encoding      = $TaggingFormat;
         $tagWriter->remove_other_tags = true;
         $tagWriter->tag_data          = $tagData;
+        /*
+        *  Currently getid3 doesn't remove pictures on *nix, only vorbiscomments.
+        *  This hasn't been tested on Windows and there is evidence that
+        *  metaflac.exe behaves differently.
+        */
+        if ($extension !== 'mp3') {
+            if (php_uname('s') == 'Linux') {
+                /* First check for installation of metaflac and
+                *  vorbiscomment system tools.
+                */
+                exec('which metaflac', $output, $retval);
+                exec('which vorbiscomment', $output, $retval1);
+
+                if ($retval !== 0 || $retval1 !== 0) {
+                    $this->logger->debug(
+                       'Metaflac and vorbiscomments must be installed to write tags to flac and oga files',
+                       [LegacyLogger::CONTEXT_TYPE => __CLASS__]
+                    );
+                    
+                    return;
+                }
+            }
+
+            if (GETID3_OS_ISWINDOWS) {
+                $command = 'metaflac.exe --remove --block-type=PICTURE ' . escapeshellarg($this->filename);
+            } else {
+                $command = 'metaflac --remove --block-type=PICTURE ' . escapeshellarg($this->filename);
+            }
+            $commandError = `$command`;
```

Comment:
Happy with everything but is this variable used? it's coming up as dead code in scrutinizer

Reply:
if you are referring to the variable `$command`, yes. 

See line 347: ``$commandError = `$command`;``    The backticks indicate that the metaflac command is being issued to the system to remove the pictures as a work-around.  It's not dead code.